hack/deploy-custom-nto.sh

```diff
@@ -31,7 +32,14 @@ cvo_scale() {
 
 nto_deploy_custom() {
   oc project openshift-cluster-node-tuning-operator
-  oc patch deploy cluster-node-tuning-operator -p '{"spec":{"template":{"spec":{"containers":[{"env":[{"name":"CLUSTER_NODE_TUNED_IMAGE","value":"'$IMAGE'"}],"name":"cluster-node-tuning-operator"}]}}}}'
+  cd "${WORKDIR}" || { echo "failed to change dir to ${WORKDIR}" >&2; exit 1; }
```

Comment:
Changing user's working directory and not changing it back where they were could be a bit annoying for them.  Can we prevent this in a clean way?

Reply:
Adding `cd -` after the for loop?

Reply:
AFAICS, we don't need to change the directory in `nto_deploy_custom()` at all, do we?  It only needs to be slightly updated.  Where we'll likely need to chagne a directory is where you're calling the `make` commands.  For that, you can use `make -C $WORKDIR`.  Then, you shouldn't need to the directory in the script at all.  Please let me know if I've missed something.